frontend/src/pages/AccountSend.vue

```diff
@@ -366,7 +366,7 @@
                       placeholder="vitalik.eth"
                       :label="$t('Send.receiver-addr-ens')"
                       lazy-rules
-                      :rules="isValidId"
+                      :rules="(value: string) => isValidId(value)"
```

Comment:
Is this change necessary? Is there a different behavior when the rules are defined this way?

Reply:
> Is this change necessary? Is there a different behavior when the rules are defined this way?

I think this is necessary because we need to pass the value of the text input object being changed into the function `isValidId` (and ultimately into `checkConfusables`). If we left it the way it was, inside `checkConfusables`.. where we do the following:
```
    const recipientIdString = recipientId.value || '';
```
We end up always getting the contents of the text object on the send page but not one of the objects on the batch send page. So we end up checking confusables on the wrong string.